pwa/src/services/getStatusColorAndIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import { faCheck, faTriangleExclamation, faXmark } from "@fortawesome/free-solid-svg-icons";
```

Comment:
Global note here: this should not be a `.tsx` file, since we're not actually writing any tsx.

Also: whenever we would be returning tsx, it's probably not a service.

Reply:
Another note: this function is very hard to use generically, but it is placed somewhere it _has_ to be generic (in the services folder).

Reason: we're accepting a prop `value: any`, no idea what this prop may or may not hold. I see some checks on if it's a string 200, if its a boolean true, if it's a string no known status, but right now I cannot send a 201 (which is also a successful call).

Thereafter, we're returning a string, which is not what we'd expect from this function, see other comments.